doc/api/vm.md

```diff
@@ -129,7 +129,7 @@ console.log(util.inspect(sandbox));
 event loops and corresponding threads being started, which have a non-zero
 performance overhead.
 
-### script.runInNewContext([sandbox][, options])
+### script.runInNewContext(sandbox[, options])
```

Comment:
@tniessen: what @mscdex suggested, and what you finally did are different. AFAICT from a quick look at the source (and knowing nothing about vm), his syntax is correct and yours is not quite.